src/Events/Controllers/EventsController.cs

```diff
@@ -147,6 +144,17 @@ private static (bool IsValid, string ErrorMessage) ValidateQueryParams(string af
             return (true, null);
         }
 
+        private static (bool IsValid, string ErrorMessage) ValidateCloudEvent(CloudEvent cloudEvent)
+        {
+            string eventResource = cloudEvent["resource"].ToString();
```

Comment:
could wait with this until we implement authorization of POST. But added it during discussions. Can remove if needed.